pytorch_pfn_extras/onnx/export_testcase.py

```diff
@@ -79,7 +87,12 @@ def strip_cmd(cmd):
     return ret
 
 
-def _export_util(model, args, f, **kwargs):
+def _export_util(
+        model: torch.nn.Module,
+        args: Tuple[Any, ...],
```

Comment:
```suggestion
        args: Sequence[Any, ...],
```

Reply:
For compatibility with [torch.onnx.export](https://pytorch.org/docs/stable/onnx.html#torch.onnx.export) maybe `Union[Sequence[Any, ...], torch.Tensor]` might be better.
Sometimes list would be also passed so allowing it with `Sequence` would be better

Reply:
Fixed in 5058f03! Thanks!